packages/clay-toolbar/README.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: 'Toolbar'
+description: 'A toolbar is a set of actions related to a specific context that are grouped into a horizontal bar.'
+lexiconDefinition: 'https://liferay.design/lexicon/core-components/toolbars/'
+packageNpm: '@clayui/toolbar'
+sibling: 'docs/components/css-toolbar.html'
+---
+
+import {ComplexToolbar, Toolbar} from '$packages/clay-toolbar/docs/index';
+
+<div class="nav-toc-absolute">
+<div class="nav-toc">
+
+-   [Introduction](#introduction)
+-   [Composing](#composing)
+-   [API](#api)
+    -   [Action](#action)
+    -   [Item](#item)
+    -   [Label](#label)
+    -   [Link](#link)
+    -   [Nav](#nav)
+    -   [Section](#section)
+
+</div>
+</div>
+
+## Introduction
+
+A toolbar organizes actions into a horizontal bar that is responsive. Its height changes to accommodate the height of the elements it contains. The toolbar always maintains the vertical alignment.
+
+<Toolbar />
+
+## Composing
+
+**Nav** is the highest level wrapper to be used inside the Toolbar to wrap all the content inside.
```

Comment:
For these bolded component names, I would prefix them, `ClayToolbar.*`, since that is the way they will be used.